srfi-207.html

```diff
@@ -171,17 +176,22 @@ <h3>Constructors</h3>
 Otherwise, an error satisfying <code>bytestring-error?</code> is signaled.</p>
 
 <p><code>(make-bytestring!</code>&nbsp;<var>bytevector at list</var><code>)</code></p>
-If the elements of <var>list</var> are suitable arguments for
-<code>bytestring</code>, writes the bytes of the bytevector that would be the
+<p>If the elements of <var>list</var> are suitable arguments for
+<code>bytestring</code>, write the bytes of the bytevector that would be the
```

Comment:
Changing "writes" to "write" makes the verb inconsistent with the other verbs in this document.  I made a pass to fix that.  Would you please undo this?

Reply:
Yes, apologies. I thought I merged all the grammar fixes from 7c0a2863 when resolving a conflict, but I guess I missed that one.